pkgs/by-name/wi/wikit/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  mkYarnPackage,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nix-update-script,
+}:
+let
+  self = mkYarnPackage {
```

Comment:
We don't use `mkYarnPackage` anymore: https://github.com/NixOS/nixpkgs/issues/324246

Reply:
Ops, my bad, I didn't find anything about the `mkYarnPackage` deprecation on https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/javascript.section.md.

I replaced it with yarn hooks.